src/scripts/main.js

```diff
@@ -1,7 +1,24 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const container = document.createElement('div');
+  const containerTitle = document.createElement('h2');
+  const containerDescr = document.createElement('p');
+
+  container.classList.add('notification', type);
+  container.style.top = `${posTop}px`;
+  container.style.right = `${posRight}px`;
+  containerTitle.classList.add('title');
+  containerTitle.innerText = title;
+  containerDescr.innerText = description;
+
+  container.appendChild(containerTitle);
+  container.appendChild(containerDescr);
+  document.body.appendChild(container);
+
+  setTimeout(() => {
```

Comment:
The removal of the notification after a timeout is a good feature. However, you should check if the container still exists in the DOM before attempting to remove it to avoid potential errors.